.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: pip install wheel
```

Comment:
Probably should specify `setuptools` too, since that's what the setup.py depends on; it happens to come with `pip`, but I don't think we should implicitly assume that.